tools/amd_build/build_amd.py

```diff
@@ -5,15 +5,20 @@
 import os
 import sys
 
+sys.path.append(
+    os.path.realpath(
+        os.path.join(__file__, os.path.pardir, os.path.pardir, os.path.pardir)
+    )
+)
+
 sys.path.append(
     os.path.realpath(
         os.path.join(
-            __file__, os.path.pardir, os.path.pardir, os.path.pardir, "torch", "utils"
-        )
+            __file__, os.path.pardir, os.path.pardir, os.path.pardir, "torch", "utils")
```

Comment:
@xw285cornell Shouldn't the closing parenthesis stay in a new line to be consistent with the code/lint style?